src/main/java/io/jenkins/plugins/ml/Server.java

```diff
@@ -74,26 +83,78 @@ public long getMaxResults() {
         public String getDisplayName() {
             return "Server";
         }
+
+        public FormValidation doCheckServerName(@QueryParameter String serverName) {
+            if( Util.fixEmptyAndTrim(serverName) == null){
+                return FormValidation.warning("* Required ");
+            }
+            String regex = "^[a-zA-Z0-9_]+$";
+            Pattern pattern = Pattern.compile(regex);
+            if( pattern.matcher(serverName).matches() ){
+                return FormValidation.ok();
+            }else{
+                return FormValidation.warning(" Try a another name. Not supported",serverName);
```

Comment:
There's an extra "a" there. Just "Try another name..." should be OK

Reply:
+1